go.mod

```
@@ -1,10 +1,11 @@
 module github.com/kopia/kopia
 
-go 1.13
+go 1.12
```

This should not be changed.
```suggestion
go 1.13
```